trove/trovesearch/page_cursor.py

```diff
@@ -14,10 +14,13 @@
 MANY_MORE = -1
 MAX_OFFSET = 9997
 
+DEFAULT_PAGE_SIZE = 13
+MAX_PAGE_SIZE = 10000
```

Comment:
Is this maximum reasonable? Looks like it was previously `101`

Reply:
update: now streams, loading only one page (~100 rows) at a time, but streaming more than ~4000 items total still times out -- can further optimize or we can talk about increasing those timeouts for responses that are actively sending data...